csv-res-filter/src/main/java/com/ibm/g11n/pipeline/CSVFilter.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.ibm.g11n.pipeline.example;
+package com.ibm.g11n.pipeline;
```

Comment:
Change package to com.ibm.g11n.pipeline.resfilter.csv

Reply:
updated - along with test classes packages